how-to/how-to-use-k8s-with-cri-containerd-and-kata.md

```diff
@@ -20,33 +18,28 @@ The Kubernetes cluster will use the
 [CRI containerd plugin](https://github.com/containerd/cri) and
 [Kata Containers](https://katacontainers.io) to launch untrusted workloads.
 
+For Kata Containers 1.5.0-rc2 and above, we will use [`containerd-shim-kata-v2` (short as `shimv2` in this documentation)](../design/shimv2.md) 
+launch Kata Containers. While for the previous version of Kata Containers, the Pods will be launched with `kata-runtime`.
```

Comment:
Suggested rewrite: 
to launch Kata Containers. For the previous version of Kata Containers, the Pods are launched with `kata-runtime`.